server/db/events.py

```diff
@@ -0,0 +1,32 @@
+"""PURPOSE: https://fastapi.tiangolo.com/advanced/events/
```

Comment:
Why do you want to make a connection to the database before it needs to even be used? What's the logic behind this?

Reply:
First, I'd like to understand how we should create that connection: with SQLAlchemy's `AsyncSessionLocal` or directly with `asyncpg` drivers to manage it. Then, I'd like to explore whether the connection should be managed through some event, via a dependency hook, or something else.

In short, I'd like to investigate the various strategies with managing connections. I created #8 for that purpose. It can be worked on in parallel to other tasks; it's not blocking anything.